### ISSUE TITLE
Add screen reader text to MTA header logo

### ISSUE THREAD

**Issue by NomeQ** [Trillium-Solutions/mendocino-wp-theme#10]
Add screen reader text within the link element that says "Mendocino Transit Authority". Make this screen reader text an `<h1>` element on just the homepage.

**Comment by NomeQ** [comment]
- Text should be inside the `<a>` element in the header, so that it applies as link text
- Using `display-none` will hide the element from screen readers as well. Create a class named "screen-reader-text" (or maybe you already have one from the skip-link element) and apply this to the page title text, so it still shows up to screen readers. The css to hide visually but not for screen readers should be in most of our newer themes, as .screen-reader-text.

**Comment by NomeQ** [comment]
`#logo-text` still has `display:none` set on it, which will hide it from screen-readers.
